.github/workflows/release-macos.yml

```diff
@@ -44,6 +44,7 @@ jobs:
 
     - name: Prepare vcpkg
       run: |
+        brew install pkg-config
```

Comment:
Maybe add it to the pandoc entry?

Reply:
Yeah makes sense, and I should also copy the dependency section to CI workflow